src/Chart/Chart.ts

```diff
@@ -137,6 +142,12 @@ export default class Chart {
 
 		$$.beforeInit();
 		$$.init();
+
+		// if ($$.config.render.lazy !== false && hasStyle($$.$el.chart, {"display": "none", "visibility": "hidden"})) {
+		// 	hook = () => {
+		// 		logError(`The call of APIs won't work. Please, make sure if chart element is %cvisible.`);
+		// 	};
+		// }
```

Comment:
shouldn't this has been removed ?

Reply:
Left for future consideration as comment.